packages/flame/lib/src/camera/viewports/fixed_resolution_viewport.dart

```diff
@@ -4,9 +4,11 @@ import 'package:flame/camera.dart';
 import 'package:flame/extensions.dart';
 import 'package:flame/src/effects/provider_interfaces.dart';
 
-/// [FixedAspectRatioViewport] is a rectangular viewport which auto-expands to
-/// take as much space as possible within the canvas, while maintaining a fixed
-/// aspect ratio.
+/// The [FixedResolutionViewport] is a rectangular viewport which auto-expands
+/// to take as much space as possible within the canvas, while maintaining a
+/// fixed resolution and aspect ratio. I.e. the viewport will always show the
+/// same portion ([resolution]) of the game world, regardless of the size of the
```

Comment:
how about "same dimensions ([resolution])"?

I actually think the name is confusing because it is not just a resolution, it will be a fixed size right

if I set it to 300x200 that doesn't mean 3 by 2, it will actually always be 300x200 logical pixels of the game world (IIRC)

of course a rename is a minor breaking change :P